pipeline/backend/docker/docker.go

```diff
@@ -52,6 +55,14 @@ func (e *docker) Load() error {
 	}
 	e.client = cli
 
+	if enableIPV6, err := strconv.ParseBool(os.Getenv("WOODPECKER_BACKEND_DOCKER_ENABLE_IPV6")); err == nil {
+		e.enableIPv6 = enableIPV6
+	} else {
+		e.enableIPv6 = false
+	}
```

Comment:
```suggestion
	e.enableIPv6, _ = strconv.ParseBool(os.Getenv("WOODPECKER_BACKEND_DOCKER_ENABLE_IPV6"))
```

if parse return err, value is already false